tests/integration/conftest.py

```diff
@@ -239,3 +244,22 @@ def finalize():
 @pytest.fixture()
 def testsroot():
     return TESTS_ROOT
+
+
+@pytest.fixture()
+def virtualenv(pathlib_tmpdir):
+    virtualenv_path = pathlib_tmpdir / "venv"
+    with temp_environ():
+        c = delegator.run("virtualenv {}".format(virtualenv_path), block=True)
+        assert c.return_code == 0
+        for name in ("bin", "Scripts"):
```

Comment:
The alternative fixture here is to use `pipenv.environment.Environment()` which provides a contextmanager called `activated`:

```python

from pipenv.environment import Environment

@pytest.fixture()
def virtualenv(pathlib_tmpdir, PipenvInstance):
    virtualenv_path = pathlib_tmpdir / "venv"
    with temp_environ():
        c = delegator.run("virtualenv {}".format(virtualenv_path), block=True)
        assert c.return_code == 0
        environment = Environment(prefix=virtualenv_path.as_posix(), is_venv=True, pipfile=PipenvInstance.pipfile)
        with environment.activated():
            yield virtualenv_path